### PR TITLE
[CON-1576] Create Integrations > SentinelOne doc

### PR DESCRIPTION
**Pull request by priyansh-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/sentinelone.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+
+**SentinelOne - Agent Overview** provides valuable insights related to SentinelOne agents deployed across an organization. The dashboard includes widgets that display key information on the total number of agents, active threats, and infected agents. It also provides information on the distribution of agents by mitigation, machine type, and scan status.
+
+Additionally, the dashboard includes information on the frequency of agents by installer type and the top 10 agents' operating systems and group summary. With this dashboard, security teams can easily monitor the performance of agents and ensure that they are adequately protecting their organization from potential threats.
+
+The SentinelOne - Agent Overview enables organizations to quickly identify any issues with agent deployment or performance and take corrective action to improve their overall security posture. It is a powerful tool for any organization looking to ensure the effectiveness and efficiency of their threat protection solutions.
+<br/><img src={useBaseUrl('img/integrations/saas-cloud/SentinelOne-Agent-Overview.png')} alt="SentinelOne-Agent-Overview.png" width="600"/>
+
+### Threat Overview Dashboard
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sentinelone.md:382]:
```suggestion
### Threat Overview
```

File: docs/integrations/saas-cloud/sentinelone.md [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+
+**SentinelOne - Threat Overview** provides a comprehensive overview of threats detected by SentinelOne, a threat intelligence and response platform. The dashboard includes widgets that display key information on unresolved threats, suspicious threats detected, threats to be mitigated, resolved threats, and threats that require a reboot. The dashboard also provides insights into the types of threats detected, their current status, and their classification source. With this dashboard, security teams can quickly identify and respond to potential security threats, minimizing the risk of security breaches and data loss.
+<br/><img src={useBaseUrl('img/integrations/saas-cloud/SentinelOne-Threat-Overview.png')} alt="SentinelOne-Threat-Overview.png" width="600"/>
+
+### Threat Analysis Dashboard
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sentinelone.md:387]:
```suggestion
### Threat Analysis
```

File: docs/integrations/saas-cloud/sentinelone.md [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+
+**SentinelOne - Threat Analysis** provides a comprehensive view of the threat landscape and enables security teams to quickly identify and respond to potential threats. The dashboard includes widgets that display key information on the confidence level of threats detected, their location across various sites, the top initiated threats, and the engines used to detect them. The dashboard also provides insights into the status of threats over time, recent incidents, and the most infected devices.
+
+With the information provided by this dashboard, security teams can effectively prioritize and manage their response to potential security threats. They can quickly identify the most critical threats, monitor the effectiveness of their threat detection engines, and respond to the most heavily infected devices. The "SentinelOne - Threat Analysis" is a valuable tool for any organization looking to proactively manage their cybersecurity risk.
+<br/><img src={useBaseUrl('img/integrations/saas-cloud/SentinelOne-Threat-Analysis.png')} alt="SentinelOne-Threat-Analysis.png" width="600"/>
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sentinelone.md:392]:
```suggestion

<img src={useBaseUrl('img/integrations/saas-cloud/SentinelOne-Threat-Analysis.png')} alt="SentinelOne-Threat-Analysis.png" width="600"/>
```

File: docs/integrations/saas-cloud/sentinelone.md [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+        "certificate",
+        "hash",
+        "path"
+    ]
+}
+```
+
+### Sample Query for Threats
+
+```sql="Suspicious threats detected"
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sentinelone.md:211]:
```suggestion
```sql title="Suspicious threats detected"
```

File: docs/integrations/saas-cloud/sentinelone.md [hunk 5]
```diff
@@ -0,0 +1,392 @@
...
+
+## Log Types
+
+The Sumo Logic App for SentinelOne consumes Threats and Agents logs, refer to the [Threats](https://usea1-partners.sentinelone.net/api-doc/api-details?category=threats&api=get-threats) and [Agents](https://usea1-partners.sentinelone.net/api-doc/api-details?category=agents&api=get-agents) documentation.
+
+### Sample Log Messages
+
+**Sample Threats Log Messages**
+
+```json
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sentinelone.md:32]:
```suggestion
```json title="Sample Threats Log Messages"
```

File: docs/integrations/saas-cloud/sentinelone.md [hunk 6]
```diff
@@ -0,0 +1,392 @@
...
+    "totalMemory": 7926,
+    "updatedAt": "2023-03-17T10:25:37.489066Z",
+    "userActionsNeeded": [],
+    "uuid": "be20d732fc87bd479e0a"
+}
+```
+
+### Sample Query for Agents
+
+```sql="Total Agents"
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sentinelone.md:334]:
```suggestion
```sql title="Total Agents"
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Formatting/style edits in comments

File: docs/integrations/saas-cloud/sentinelone.md (hunk 5 above)
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md (hunk 6 above)
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md [hunk 7]
```diff
@@ -0,0 +1,392 @@
+---
+id: sentinelone
+title: Sumo Logic App for SentinelOne
+sidebar_label: SentinelOne
+description: The Sumo Logic App for the SentinelOne App offers functionality for monitoring and analyzing your organization's SentinelOne Threats and Agents logs.
```
[the remark this answers is not in the document]
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md [hunk 8]
```diff
@@ -0,0 +1,392 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/sentinelone-icon.png')} alt="sentinelone-icon.png" width="100" />
+
+The SentinelOne App for Sumo Logic is designed to provide security professionals with a comprehensive view of their organization's security posture. Based on Sumo Logic’s SentinelOne Source, this app allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.
+
+With the SentinelOne App, you can gain valuable insights into your endpoint security data, including threat intelligence, endpoint activity, and system health. You can monitor your endpoints for suspicious behaviour and quickly respond to any security incidents that may arise. 
+The app provides pre-built dashboards that enable you to quickly visualize your SentinelOne data and gain insights into your organization's security posture. You can customise these dashboards to meet your specific needs and monitor the metrics that matter most.
+With the SentinelOne App for Sumo Logic, you can:
```
[the remark this answers is not in the document]
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md (hunk 1 above)
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md (hunk 2 above)
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md (hunk 4 above)
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md (hunk 3 above)
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md [hunk 9]
```diff
@@ -0,0 +1,392 @@
...
+| where agentOsType matches "{{os}}" or isNull(agentOsType)
+| where agentMachineType matches "{{machine}}" or isNull(agentMachineType)
+| where confidenceLevel matches "{{confidence_level}}" or isNull(confidenceLevel)
+| where confidenceLevel = "suspicious"
+| count_distinct(id)
+```
+
+**Sample Agents Log Messages**
+
+```json
```
[the remark this answers is not in the document]
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md [hunk 10]
```diff
@@ -0,0 +1,392 @@
...
+
+## Viewing SentinelOne Dashboards​
+
+* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
+
+ You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
+
+* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+
+### Agent Overview Dashboard
```
[the remark this answers is not in the document]
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/sentinelone.md]: Resolved

File: docs/integrations/saas-cloud/sentinelone.md [hunk 11]
```diff
@@ -0,0 +1,395 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/sentinelone-icon.png')} alt="sentinelone-icon.png" width="100" />
+
+The SentinelOne App for Sumo Logic is designed to provide security professionals with a comprehensive view of their organization's security posture. Based on Sumo Logic’s SentinelOne Source, this app allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.
+
+With the SentinelOne App, you can gain valuable insights into your endpoint security data, including threat intelligence, endpoint activity, and system health. You can monitor your endpoints for suspicious behavior and quickly respond to any security incidents that may arise. 
+
+The app provides pre-built dashboards that enable you to quickly visualize your SentinelOne data and gain insights into your organization's security posture. You can customize these dashboards to meet your specific needs and monitor the metrics that matter most. With the SentinelOne App for Sumo Logic, you can:
+* Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response
```
Comment (JainM6) [inline docs/integrations/saas-cloud/sentinelone.md:17]:
```suggestion
* Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response.
```

File: docs/integrations/saas-cloud/sentinelone.md [hunk 12]
```diff
@@ -0,0 +1,395 @@
...
+* Gain valuable insights into your organization's security posture with pre-built dashboards
+* Customize dashboards to meet your specific needs and monitor the metrics that matter most to you
+
+In summary, the SentinelOne App for Sumo Logic provides security professionals with the tools they need to monitor and respond to security incidents in real-time, ensuring the protection of their organization's critical assets.
+
+## Log Types
+
+The Sumo Logic App for SentinelOne consumes Threats and Agents logs, refer to the [Threats](https://usea1-partners.sentinelone.net/api-doc/api-details?category=threats&api=get-threats) and [Agents](https://usea1-partners.sentinelone.net/api-doc/api-details?category=agents&api=get-agents) documentation.
+
+### Sample Log Messages
```
Comment (JainM6) [inline docs/integrations/saas-cloud/sentinelone.md:28]: It'd be great if we could write both log messages under the Sample Log Messages heading and queries under the Sample Queries section.

File: docs/integrations/saas-cloud/sentinelone.md [hunk 13]
```diff
@@ -0,0 +1,395 @@
...
+
+### Agent Overview
+
+**SentinelOne - Agent Overview** provides valuable insights related to SentinelOne agents deployed across an organization. The dashboard includes widgets that display key information on the total number of agents, active threats, and infected agents. It also provides information on the distribution of agents by mitigation, machine type, and scan status.
+
+Additionally, the dashboard includes information on the frequency of agents by installer type and the top 10 agents' operating systems and group summary. With this dashboard, security teams can easily monitor the performance of agents and ensure that they are adequately protecting their organization from potential threats.
+
+The SentinelOne - Agent Overview enables organizations to quickly identify any issues with agent deployment or performance and take corrective action to improve their overall security posture. It is a powerful tool for any organization looking to ensure the effectiveness and efficiency of their threat protection solutions.
```
Comment (JainM6) [inline docs/integrations/saas-cloud/sentinelone.md:379]:
```suggestion
The SentinelOne Agent Overview enables organizations to quickly identify any issues with agent deployment or performance and take corrective action to improve their overall security posture. It is a powerful tool for any organization looking to ensure the effectiveness and efficiency of its threat protection solutions.
```

**Review by JainM6 — DISMISSED** [review]
Minor Heading fixes

File: docs/integrations/saas-cloud/sentinelone.md [hunk 14]
```diff
@@ -0,0 +1,395 @@
...
+title: Sumo Logic App for SentinelOne
+sidebar_label: SentinelOne
+description: The Sumo Logic App for SentinelOne offers functionality for monitoring and analyzing your organization's SentinelOne Threats and Agents logs.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/sentinelone-icon.png')} alt="sentinelone-icon.png" width="100" />
+
+The SentinelOne App for Sumo Logic is designed to provide security professionals with a comprehensive view of their organization's security posture. Based on Sumo Logic’s SentinelOne Source, this app allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sentinelone.md:12]:
```suggestion
The SentinelOne App for Sumo Logic provides security professionals with a comprehensive view of their organization's security posture. The app, based on Sumo Logic’s SentinelOne Source, allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.
```

File: docs/integrations/saas-cloud/sentinelone.md [hunk 15]
```diff
@@ -0,0 +1,395 @@
...
+| where siteName matches "{{site}}" or isNull(siteName)
+| where mitigationStatus matches "{{mitigation_status}}" or isNull(mitigationStatus)
+| where agentOsType matches "{{os}}" or isNull(agentOsType)
+| where agentMachineType matches "{{machine}}" or isNull(agentMachineType)
+| where confidenceLevel matches "{{confidence_level}}" or isNull(confidenceLevel)
+| where confidenceLevel = "suspicious"
+| count_distinct(id)
+```
+
+**Sample Agents Log Messages**
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sentinelone.md:223]: Don't need this (title is included in the snippet below)
```suggestion
```

**Review by kimsauce — APPROVED** [review]

File: docs/integrations/saas-cloud/sentinelone.md [hunk 16]
```diff
@@ -0,0 +1,393 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/sentinelone-icon.png')} alt="sentinelone-icon.png" width="100" />
+
+The SentinelOne App for Sumo Logic provides security professionals with a comprehensive view of their organization's security posture. The app, based on Sumo Logic’s SentinelOne Source, allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.
+
+With the SentinelOne App, you can gain valuable insights into your endpoint security data, including threat intelligence, endpoint activity, and system health. You can monitor your endpoints for suspicious behavior and quickly respond to any security incidents that may arise. 
+
+The app provides pre-built dashboards that enable you to quickly visualize your SentinelOne data and gain insights into your organization's security posture. You can customize these dashboards to meet your specific needs and monitor the metrics that matter most. With the SentinelOne App for Sumo Logic, you can:
+* Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response.
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/sentinelone.md:17]:
```suggestion
* Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response
```

**Review by jpipkin1 — DISMISSED** [review]
That's a very clean article! I only found one minor thing you need to fix.

File: docs/integrations/saas-cloud/sentinelone.md (hunk 12 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/sentinelone.md]: Agreed! @priyansh-sumo , I've just fixed this

**Pull request merged**